src/gui/properties/peerlistwidget.cpp

```diff
@@ -108,7 +108,7 @@ PeerListWidget::PeerListWidget(PropertiesWidget *parent)
     //its size is 0, because explicitly 'showing' the column isn't enough
     //in the above scenario.
     for (unsigned int i = 0; i < PeerListDelegate::IP_HIDDEN; i++)
-        if (!columnWidth(i))
+        if (!columnWidth(i) && !isColumnHidden(i))
```

Comment:
Can you also replace `!columnWidth(i)` to `(columnWidth(i) <= 0)`, it doesn't make sense to use logical operator on a `int`.
ps. There are 2 instances of it.